### ISSUE TITLE
CI fails to connect to the database

### ISSUE THREAD

**Issue by chr-schr** [ElektraInitiative/PermaplanT#1306]
### Describe the Bug

Recently there have been a lot of cases where CI jobs failed because the connection to the DB could not be established:
https://build.libelektra.org/blue/organizations/jenkins/PermaplanT/detail/PR-804/45/pipeline/
https://build.libelektra.org/blue/organizations/jenkins/PermaplanT/detail/PR-1291/10/pipeline/
https://build.libelektra.org/blue/organizations/jenkins/PermaplanT/detail/PR-1300/5/pipeline/

And sometimes it fails on the md-book job, but does not sound related from the error message:
https://build.libelektra.org/blue/organizations/jenkins/PermaplanT/detail/PR-1303/7/pipeline

### Steps to Reproduce the Problem

Let jenkins rebuild 

### Expected Result

CI runs through or fails at new code

### Actual Result

CI fails to connect to the database

### PermaplanT version

master

### Operating System

Windows

### Browser

Chrome

**Comment by markus2330** [comment]
Thank you for creating this issue!

Unfortunately @filo14 is on holidays. Maybe @lukashartl can help out here?

**Comment by lukashartl** [comment]
According to 

https://build.libelektra.org/blue/organizations/jenkins/PermaplanT/detail/PR-804/45/pipeline/

it fails while waiting for the database to become available.

Therefore it uses the following function:

```groovy
def wait_for_db(String command) {
    retry(6) {
        sleep(time: 5, unit: 'SECONDS')
        sh "${command}"
    }
}
```

This means that we wait 6 * 5 seconds for the database to start.

In the log output we can see, that it fails with the following errors:

```
db:5432 - no response
db:5432 - no response
db:5432 - no response
db:5432 - no response
db:5432 - no response
db:5432 - rejecting connections
```

I think this is only a timing issue. As the DB is not available in the first 5 tries (database not listining yet -- probably doing initialization), and the last time is probably due to the service being started, but the database is not accepting connection *yet*.

My first try would be to increase the retry count for another 2 tries (10 seconds).

**Comment by markus2330** [comment]
Hopefully fixed with 0e88a69f5b29f28c5c61513e388731dd4e10234d, please reopen if it still occurs.
